<!DOCTYPE html>
<html lang="en-US" prefix="og: http://ogp.me/ns#">
<head>
	<meta charset="UTF-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=10" />
	<title>Translational Bioinformatics - Robinson Lab</title>
	
<!-- This site is optimized with the Yoast SEO plugin v3.4.2 - https://yoast.com/wordpress/plugins/seo/ -->
<meta name="description" content="The Robinson Lab at the Jackson Laboratory for Genomic Medicine develops algorithms and computational resources for understanding human disease based on phenotyopes, model organisms, and genomics."/>
<link rel="canonical" href="https://robinsongroup.github.io/" />
<meta property="og:locale" content="en_US" />
<meta property="og:type" content="article" />
<meta property="og:title" content="Translational Bioinformatics - Robinson Lab"/>
<meta property="og:description" content="The Robinson Lab at the Jackson Laboratory for Genomic Medicine develops algorithms and computational resources for understanding human disease based on phenotyopes, model organisms, and genomics." />
<meta property="og:url" content="https://robinsongroup.github.io/" />
<meta property="og:site_name" content="Robinson Lab" />
<meta property="article:section" content="Uncategorized" />
<meta property="article:published_time" content="2017-01-06T02:27:05+00:00" />
<meta property="article:modified_time" content="2017-01-07T02:39:47+00:00" />
<meta property="og:updated_time" content="2017-01-07T02:39:47+00:00" />
<meta property="og:image" content="https://robinsongroup.github.io/images/phenoviz.png " />
<meta name="twitter:card" content="summary" />
<meta name="twitter:description" content="The Robinson Lab at the Jackson Laboratory for Genomic Medicine develops algorithms and computational resources for understanding human disease based on phenotyopes, model organisms, and genomics." />
<meta name="twitter:title" content="Translational Bioinformatics - Robinson Lab" />
<meta name="twitter:image"  content="https://robinsongroup.github.io/images/phenoviz.png" />
<!-- / Yoast SEO plugin. -->

<link rel='dns-prefetch' href='//ajax.googleapis.com' />
<link rel='dns-prefetch' href='//s.w.org' />
<link rel='stylesheet' id='siteorigin-panels-front-css'  href='css/front.css' type='text/css' media='all' />
<link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
<link rel='stylesheet' id='sow-button-base-css'  href='css/buttonstyle.css' type='text/css' media='all' />
<link rel='stylesheet' id='vantage-style-css'  href='css/vantage.css' type='text/css' media='all' />
<link rel='stylesheet' id='font-awesome-css'  href='css/font-awesome.css' type='text/css' media='all' />
<script type='text/javascript' src='js/jquery.js'></script>
<script type='text/javascript' src='js/jquery-migrate.min.js'></script>
<script type='text/javascript' src='js/core.min.js?ver=1.11.4'></script>

<script type='text/javascript' src='https://robinsongroup.github.io/js/jquery.flexslider.min.js?ver=2.1'></script>
<script type='text/javascript' src='https://robinsongroup.github.io/js/jquery.touchSwipe.min.js?ver=1.6.6'></script>
<script type='text/javascript' src='https://robinsongroup.github.io/js/jquery.theme-main.min.js?ver=1.4.4'></script>
<script type='text/javascript' src='https://robinsongroup.github.io/js/jquery.fitvids.min.js?ver=1.0'></script>

<meta name="viewport" content="width=device-width, initial-scale=1" />	<style type="text/css" media="screen">
  #footer-widgets .widget { width: 100%; }
  @media screen and (max-width: 640px) {
  #footer-widgets .widget { width: auto; float: none; }
  }
</style>
<meta name="twitter:partner" content="tfwp" /><style type="text/css" media="all" id="siteorigin-panels-grids-wp_head">/* Layout 1 */ #pg-1-0 .panel-grid-cell { float:none } #pl-1 .panel-grid-cell .so-panel { margin-bottom:35px } #pl-1 .panel-grid-cell .so-panel:last-child { margin-bottom:0px } #pg-1-0 { margin-left:-15px;margin-right:-15px } #pg-1-0 .panel-grid-cell { padding-left:15px;padding-right:15px } @media (max-width:780px){ #pg-1-0 .panel-grid-cell { float:none;width:auto } #pl-1 .panel-grid , #pl-1 .panel-grid-cell {  }  } </style>

</head>

<body class="post-template-default single single-post postid-1 single-format-standard siteorigin-panels responsive layout-full no-js has-sidebar has-menu-search panels-style-force-full">


<div id="page-wrapper">

	
	<header id="masthead" class="site-header" role="banner">
<div class="hgroup full-container">
	    <a href="https://robinsongroup.github.io/" title="Robinson Lab" rel="home" class="logo"><img src="images/Logo_600dpi.jpg"  class="logo-height-constrain" height="120"  alt="Robinson Lab Logo"  /> 	 </a>
	    <div class="support-text"> <span style="font-family:Times,Serif;font-size:60px;font-variant: small-caps;color:white;display:inline-block;"> 
	    </br>  &nbsp; Robinson Lab
	  </span>
	</div>
 </div><!-- .hgroup.full-container -->
<nav role="navigation" class="site-navigation main-navigation primary use-sticky-menu">

	<div class="full-container">
							<div id="search-icon">
				<div id="search-icon-icon"><div class="vantage-icon-search"></div></div>
				
			</div>
		
							<div class="menu-primary-menu-container"><ul id="menu-primary-menu" class="menu"><li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-44"><a href="https://robinsongroup.github.io/"><span class="icon"></span>Home</a></li>

<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-42"><a href="https://robinsongroup.github.io/research.html">Research</a>
<ul class="sub-menu">
  <li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-347"><a href="https://robinsongroup.github.io/research.html#phenotype">Computational Phenotype Analysis</a></li>
  <li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-347"><a href="https://robinsongroup.github.io/research.html#genomics">Bioinformatics for Genomics</a></li>
   <li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-347"><a href="https://robinsongroup.github.io/research.html#genes">NGS-based Disease-Associated Gene Discovery</a></li>
</ul>
</li>
<li id="menu-item-40" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-40"><a href="https://robinsongroup.github.io/people.html">People</a>
<ul class="sub-menu">
	<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-207"><a href="https://robinsongroup.github.io/people.html"><span class="icon"></span>Meet Our Group</a></li>
	<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-208"><a href="https://robinsongroup.github.io//people.html#jax"><span class="icon"></span>JAX</a></li>
	<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-209"><a href="https://robinsongroup.github.io/people.html#berlin"><span class="icon"></span>Alumni: Pre-2017 team in Berlin</a></li>
	<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-261"><a href="https://robinsongroup.github.io/people.html#students"><span class="icon"></span>Students</a></li>
</ul>
</li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-41"><a href="https://robinsongroup.github.io/publications.html">Publications</a></li>

<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-39"><a href="https://robinsongroup.github.io/software.html">Software</a>
<ul class="sub-menu">
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-378"><a href="https://robinsongroup.github.io/software.html">Software</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#hpo">Human Phenotype Ontology (HPO)</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#exomiser">Exomiser</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#phenomizer">Phenomizer</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#phenix">PhenIX</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#phenogramviz">PhenogramViz</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#chipseq">ChIP-seq software</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#jannovar">Jannovar</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#ontologizer">Ontologizer</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#imseq">IMSEQ</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#boqa">BOQA</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#cdn">Common Disease Network</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-377"><a href="https://robinsongroup.github.io/software.html#exomewalker">ExomeWalker</a></li>
</ul>
</li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-352"><a href="https://robinsongroup.github.io/contact.html">Contact Us</a>
</li>
</ul></div>			</div>
</nav><!-- .site-navigation .main-navigation -->
</header><!-- #masthead .site-header -->
	
	
	
	<div id="main" class="site-main">
		<div class="full-container">
			
<div id="primary" class="content-area">
	<div id="content" class="site-content" role="main">

	
		
			
<article id="post-26" class="post-26 page type-page status-publish hentry post">

	<div class="entry-main">

		
		<header class="entry-header">
			<a href="#software"><h1 class="entry-title">Software</h1></a>
			<div class="entry-content">
			  <p>The Robinson lab develops a wide range of algorithms, computational resources, and applications.</p>

			  <a href="#hpo"><h2>The Human Phenotype Ontology (HPO)</h2></a>
			    The Human Phenotype Ontology (HPO) aims to provide a standardized vocabulary of phenotypic abnormalities encountered in human disease.
			    Terms in the HPO describe individual phenotypic abnormalities such as atrial septal defect.
			    For further details and information please refer to the <a href="http://www.human-phenotype-ontology.org" target="_hpo">Human Phenotype Ontology Homepage</a>. The HPO is developed together with
			  the Berlin <a href="https://phenomics.github.io/" target="_blank">Phenomics Team</a> as a part of the <a href="http://monarchinitiative.org" target="_blank">Monarch Initiative</a>.
			  <p>Köhler S, Vasilevsky NA, Engelstad M, Foster E, McMurry J, Aymé S, Baynam G,
Bello SM, Boerkoel CF, Boycott KM, Brudno M, Buske OJ, Chinnery PF, Cipriani V,
Connell LE, Dawkins HJ, DeMare LE, Devereau AD, de Vries BB, Firth HV, Freson K, 
Greene D, Hamosh A, Helbig I, Hum C, Jähn JA, James R, Krause R, F Laulederkind
SJ, Lochmüller H, Lyon GJ, Ogishima S, Olry A, Ouwehand WH, Pontikos N, Rath A,
Schaefer F, Scott RH, Segal M, Sergouniotis PI, Sever R, Smith CL, Straub V,
Thompson R, Turner C, Turro E, Veltman MW, Vulliamy T, Yu J, von Ziegenweidt J,
Zankl A, Züchner S, Zemojtel T, Jacobsen JO, Groza T, Smedley D, Mungall CJ,
Haendel M, <b>Robinson PN</b> (2017) The Human Phenotype Ontology in 2017. <em>Nucleic Acids Res.</em> <strong>45(D1)</strong>:D865-D876. <br/>
		    <a href="http://www.ncbi.nlm.nih.gov/pubmed/27899602" target="_blank">PubMed</a></p>
	</div>
	<div class="entry-content">
			

	  <a href="#exomiser"><h2>Exomiser</h2></a>
	  The Exomiser is a Java program that functionally annotates and prioritises variants from whole-exome sequencing data starting from a 
	  VCF file. The Exomiser was developed by our group, Damian Smedley and Jules Jacobsen of the Mouse Informatics Group at the Sanger Institute (now at the 100,000 Genomes Project and Quenn Mary's University in London),
	  and other members of the <a href="http://monarchinitiative.org" target="_blank">Monarch Initiative</a>.
	  The Exomiser is available for download, and an online demo version is available <a href="http://www.sanger.ac.uk/resources/software/exomiser/" target="_blank">here</a>.
  <p>Smedley D, Jacobsen JO, <b>Jäger M</b>, <b>Köhler S</b>, <b>Holtgrewe M</b>, <b>Schubach M</b>, Siragusa E, <b>Zemojtel T</b>, Buske OJ, Washington NL, Bone WP, Haendel MA, <b>Robinson PN</b>. (2015). Next-generation diagnostics and disease-gene discovery with the Exomiser. <em>Nature Protocols</em> <strong>10</strong>:2004-15. <br/>
		    <a href="http://www.ncbi.nlm.nih.gov/pubmed/26562621" target="_blank">PubMed</a></p>

	</div>

	<div class="entry-content">
			

	  <a href="#phenomizer"><h2>Phenomizer</h2></a>
	  <p>The <a href="http://compbio.charite.de/phenomizer" target="_blank">Phenomizer</a>  aims to help clinicians to identify the correct differential diagnosis in the field of human genetics.
The user enters the signs/symptoms of the patient encoded as terms from the <a href="http://www.human-phenotype-ontology.org" target="_hpo">Human Phenotype Ontology Homepage</a>. 
The software then ranks all diseases from OMIM, Orphanet, and DECIPHER by a score that reflects how well the phenotypic profiles of the patient
	    and the disease match to each other.</p>

	  <p><b>Kohler, S.</b>, Schulz, M. H., <b>Krawitz, P.</b>, <b>Bauer, S.</b>, Dolken, S., Ott, C. E., Mundlos, C., Horn, D., Mundlos, S., and <b>Robinson, P. N.</b> (2009). Clinical diagnostics in human genetics with semantic similarity searches in ontologies. <em>Am. J. Hum. Genet.</em> 85, 457–464.<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/19800049" target="_blank">PubMed</a>  
</p>

</div>




	<div class="entry-content">
			

	  <a href="#phenix"><h2>PhenIX</h2></a>
	  PhenIX, Phenotypic Interpretation of eXomes, is a pipeline for ranking (prioritizing) candidate genes in exomes or 
NGS panels with comprehensive coverage of human Mendelian disease genes. It ranks genes based on predicted variant 
pathogenicity as well as phenotypic similarity of diseases associated with the genes harboring these variants to the 
phenotypic profile of the individual being investigated, based on analysis 
powered by the Human Phenotype Ontology (HPO). 
	  An online demo version of PhenIX is available <a href="http://compbio.charite.de/PhenIX/" target="_blank">here</a>.

	  <p>Zemojtel, T., <b>Kohler, S.</b>, Mackenroth, L., <b>Jager, M.</b>, Hecht, J., <b>Krawitz, P.</b>, Graul-Neumann, L., Doelken, S., Ehmke, N., Spielmann, M., et al. (2014). Effective diagnosis of genetic disease by computational phenotype analysis of the disease-associated genome. <em>Sci Transl Med</em> <strong>6</strong>:252ra123.<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25186178" target="_blank">PubMed</a>  
</p>

	</div>

	
	<div class="entry-content">
			

	  <a href="#phenogramviz"><h2>PhenogramViz</h2></a>
	  PhenogramViz is a tool that automatically analyses and visualizes gene-to-phenotype relations for a set of genes affected by CNV of a patient and a set of HPO-terms representing the symptoms of said patient.
	  The tool makes full use of the cross-species phenotype ontology <a href="http://f1000research.com/articles/2-30/v2" target="_blank">uberpheno</a>.
	  Video Tutorials are available on the PhenogramViz <a href="https://www.youtube.com/channel/UC-hLx_KWD5zlbf7VnV3-fBA" target="_blank">Youtube-Channel</a>.

	  <p><b>K&ouml;hler, S.</b>, Schoeneberg, U., Czeschik, J. C., Doelken, S. C., Hehir-Kwa, J. Y., <b>Ibn-Salem, J.</b>, Mungall, C. J., Smedley, D., Haendel, M. A., and <b>Robinson, P. N.</b> (2014). Clinical interpretation of CNVs with cross-species phenotype data. <em>J. Med. Genet.</em> <strong>51</strong>:766–772.<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25280750" target="_blank">PubMed</a>  
</p>	</div>

<div class="entry-content">
			

	  <a href="#chipseq"><h2>ChIP-seq software</h2></a>
	  Chromatin immunoprecipitation coupled with next-generation sequencing (ChIP-seq) is a powerful technology to identify the genome-wide locations of transcription factors and other DNA binding proteins. Computational ChIP-seq peak calling infers the location of protein-DNA interactions based on various measures of enrichment of sequence reads.
	  <p>Our algorithm, Q,  uses an assessment of the quadratic enrichment of reads to center candidate peaks followed by statistical analysis of saturation of candidate peaks by 5' ends of reads. We show that our method not only is substantially faster than several competing methods but also demonstrates statistically significant advantages with respect to reproducibility of results and in its ability to identify peaks with reproducible binding site motifs.</p>

	  <p><b>Hansen P</b>, <b>Hecht J</b>, Ibrahim DM, Krannich A, Truss M, <b>Robinson PN</b> (2015) Saturation
analysis of ChIP-seq data for reproducible identification of binding peaks.
  <em>Genome Res</em> <strong>25</strong>:1391-400. </br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/26163319" target="_blank">PubMed</a>  
	  </p>

	  ChIP-exo and ChIP-nexus are related methods that can characterize genomic binding sites of proteins with higher resolution than standard ChIP-seq, but come with their own methodological and statistical challenges. We developed an algorithm for quickly evaluation ChIP-exo and ChIP-nexus datasets called Q-nexus.

	   <p>Hansen P, Hecht J, Ibn-Salem J, Menkuec BS, Roskosch S, Truss M, <b>Robinson PN</b> (2016)
Q-nexus: a comprehensive and efficient analysis pipeline designed for ChIP-nexus. <em>BMC Genomics</em> <strong>17</strong>:873 <br/>
		    <a href="http://www.ncbi.nlm.nih.gov/pubmed/" target="_blank">PubMed</a></p>


</div>
<div class="entry-content">

<a href="#jannovar"><h2>Jannovar</h2></a>
<h3>A Java library for Exome Annotation</h3>
Transcript-based annotation and pedigree analysis are two basic steps in the computational analysis of whole-exome sequencing experiments in disease-gene discovery projects or diagnostics. Jannovar is a stand-alone Java application as well as a Java library designed to be used in larger software frameworks for exome analysis. Jannovar uses an interval tree to identify all transcripts affected by a given variant, and provides HGVS-compliant annotations for both for variants affecting coding sequences and splice junctions as well as UTR sequences and non-coding RNA transcripts. Jannovar can also perform family-based pedigree analysis with VCF files with data from members of a family segregating a Mendelian disorder. Using a desktop computer, Jannovar requires a few seconds to annotate a typical VCF file with exome data.
<h3>Downloads</h3>
For convenience, we provide a compiled version of the Jannovar executable. The latest version of jannovar is always available at the <a href="https://github.com/charite/jannovar/releases" target="_blank">project github repository</a>. See the instructions in the
<a href="http://jannovar.readthedocs.org/en/master/" target="_blank">online manual</a> for information about how to use Jannovar.
<h3>Downloading the source code</h3>
Developers can download the entire source code for the Jannovar project in the form of a maven repository from the <a href="https://github.com/charite/jannovar" target="_blank">GitHub page</a>.
<h3>Current version</h3>
The version of Jannovar originally published in 2014 has been substantially extended by Manuel Holtgrewe and Max Schubach. Many of the new features are described in the <a href="http://jannovar.readthedocs.org/en/master/" target="_blank">online manual</a>.


<p><b>Jager, M.</b>, Wang, K., <b>Bauer, S.</b>, Smedley, D., <b>Krawitz, P.</b>, and <b>Robinson, P. N.</b> (2014). Jannovar: a java library for exome annotation. <em>Hum. Mutat.</em> 35, 548–555.<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/24677618" target="_blank">PubMed</a>  
</p>

	</div>
 <div class="entry-content">

<div class="entry-content">

<a href="#ontologizer"><h2>Ontologizer</h2></a>
<h3>Analyze and Visualize High-Throughput Biological Data Using Gene Ontology</h3>


The Ontologizer is a tool for the statistical analysis and visualization of 
high-throughput biological data using Gene Ontology. The Ontologizer can be started as a Java Webstart app from the <a href="[http://ontologizer.de/" target="_blank">project's homepage</a>.

<p><b>Bauer, S.</b>, Grossmann, S., Vingron, M., and <b>Robinson, P. N.</b> (2008). Ontologizer 2.0–a multifunctional tool for GO term enrichment analysis and data exploration. <em>Bioinformatics</em> 24, 1650–1651.<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/18511468" target="_blank">PubMed</a>  
</p>

<p><b>Bauer, S.</b>, <b>Robinson, P. N.</b>, and Gagneur, J. (2011). Model-based gene set analysis for Bioconductor. <em>Bioinformatics</em> 27, 1882–1883.<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/21561920" target="_blank">PubMed</a>  
</p>


	</div>
<div class="entry-content">

  <div class="entry-content">

<a href="#imseq"><h2>IMSEQ</h2></a>
<h3>IMSEQ--a fast and error aware approach to immunogenetic sequence analysis</h3>

<p>Recombined T- and B-cell receptor repertoires are increasingly being studied using next generation sequencing (NGS) in order to interrogate the repertoire composition as well as changes in the distribution of receptor clones under different physiological and disease states. This type of analysis requires efficient and unambiguous clonotype assignment to a large number of NGS read sequences, including the identification of the incorporated V and J gene segments and the CDR3 sequence. Current tools have deficits with respect to performance, accuracy and documentation of their underlying algorithms and usage.</p>
<p>IMSEQ is a method to derive clonotype repertoires from NGS data with sophisticated routines for handling errors stemming from PCR and sequencing artefacts. The application can handle different kinds of input data originating from single- or paired-end sequencing in different configurations and is generic regarding the species and gene of interest.</p>
<p>The software can be downloaded from the <a href="www.imtools.org" target="_blank">project homepage</a>.</p>
<p><b>Kuchenbecker, L.</b>, Nienen, M., Hecht, J., Neumann, A. U., Babel, N., Reinert, K., and <b>Robinson, P. N.</b> (2015). IMSEQ - a fast and error aware approach to immunogenetic sequence analysis. <em>Bioinformatics</em>, .<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25987567" target="_blank">PubMed</a>  
</p>

	</div>
 

     <div class="entry-content">

<a href="#boqa"><h2>BOQA</h2></a>
<h3>BOQA - The Bayesian Ontology Query Algorithm</h3>

<p>BOQA integrates the knowledge stored in an ontology and the accompanying annotations into a Bayesian network in order to implement a search system in which users enter one or more terms of the ontology to get a list of appropriate domain items. This is the companion Website to the original Bioinformatics publication. Here, we provide an implementation as well as the Benchmark procedure. The source code can be downloaded <a href="http://compbio.charite.de/tl_files/boqa/boqa-dist.tar.gz">here</a>.
</p>
<p><b>Bauer, S.</b>, <b>Kohler, S.</b>, Schulz, M. H., and <b>Robinson, P. N.</b> (2012). Bayesian ontology querying for accurate and noise-tolerant semantic searches. <em>Bioinformatics</em> 28, 2502–2508.<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/22843981" target="_blank">PubMed</a>  
</p>

     </div>


        <div class="entry-content">

<a href="#cdn"><h2>Common Disease Network</h2></a>
<h3>A web-app to explore common diseases</h3>

<p>We have developed a <a href="http://compbio.charite.de/CDN/cdn.html">web application</a> that allows interactive exploration of a network of common diseases. The edges between the common diseases are based on similarity of the phenotypic profiles of the two diseases. If the similarity exceeds a threshold, we draw an edge, otherwise not. The similarity is calculated using the [Human Phenotype Ontology](http://human-phenotype-ontology.org).

  <p>Groza, T., <b>K&ouml;hler, S.</b>, Moldenhauer, D., Vasilevsky, N., Baynam, G., Zemojtel, T., Schriml, L. M., Kibbe, W. A., Schofield, P. N., Beck, T., et al. (2015). The Human Phenotype Ontology: Semantic Unification of Common and Rare Disease. <em>The American Journal of Human Genetics</em> <strong>97</strong>:111-24.<br/>   <a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00234-7" target="_blank">PubMed</a>  
</p>

	</div>

	
        <div class="entry-content">

<a href="#exomewalker"><h2>ExomeWalker</h2></a>
<h3>A web-app to explore common diseases</h3>

<p>ExomeWalker is a computational method to prioritise a set of candidates in exome sequencing projects that aim to identify novel Mendelian disease genes. 
  Genes are prioritised according to a variant score (predicted pathogenicity, rarity, pattern of variants compatible with the assumed mode of inheritance) and to their vicinity to other genes that belong to the same disease-gene family within the protein protein interaction (PPI) network, using the Random-Walk method as described previously to determine similarity within the PPI network on the basis of the global characteristics of the network.</p>
<p>ExomeWalker can be run as a part of the Exomiser suite. An online demo version will be put online again in the course of 2017.</p>
<p>Smedley, D., <b>Kohler, S.</b>, Czeschik, J. C., Amberger, J., Bocchini, C., Hamosh, A., Veldboer, J., Zemojtel, T., and <b>Robinson, P. N.</b> (2014). Walking the interactome for candidate prioritization in exome sequencing studies of Mendelian diseases. <em>Bioinformatics</em> 30, 3215–3222.<br/>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25078397" target="_blank">PubMed</a>  
</p>

	</div>
  </div> <!--entry-content -->




</article><!-- #post-26 -->



</div><!-- #content .site-content -->
</div><!-- #primary .content-area -->



					</div><!-- .full-container -->
	</div><!-- #main .site-main -->

	
	
	<footer id="colophon" class="site-footer" role="contentinfo">

	<div id="footer-widgets" class="full-container">
			</div><!-- #footer-widgets -->
</footer><!-- #colophon .site-footer -->
	
</div><!-- #page-wrapper -->


<a href="#" id="scroll-to-top" title="Back To Top"><span class="vantage-icon-arrow-up"></span></a><script type='text/javascript' src='js/datepicker.min.js?ver=1.11.4'></script>


</body>
</html>
