
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://robinsongroup.github.io/software/">
      
      
        <link rel="prev" href="../publications/">
      
      
        <link rel="next" href="../teaching/teaching/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.18">
    
    
      
        <title>Software - Robinson Lab</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e37652d.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/badge.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
      
        <meta  property="og:type"  content="website" >
      
        <meta  property="og:title"  content="Software - Robinson Lab" >
      
        <meta  property="og:description"  content="None" >
      
        <meta  property="og:image"  content="https://robinsongroup.github.io/assets/images/social/software.png" >
      
        <meta  property="og:image:type"  content="image/png" >
      
        <meta  property="og:image:width"  content="1200" >
      
        <meta  property="og:image:height"  content="630" >
      
        <meta  property="og:url"  content="https://robinsongroup.github.io/software/" >
      
        <meta  name="twitter:card"  content="summary_large_image" >
      
        <meta  name="twitter:title"  content="Software - Robinson Lab" >
      
        <meta  name="twitter:description"  content="None" >
      
        <meta  name="twitter:image"  content="https://robinsongroup.github.io/assets/images/social/software.png" >
      
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="teal" data-md-color-accent="purple">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#software" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Robinson Lab" class="md-header__button md-logo" aria-label="Robinson Lab" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Robinson Lab
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Software
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="teal" data-md-color-accent="purple"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6m0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4M7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="teal" data-md-color-accent="lime"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../publications/" class="md-tabs__link">
        
  
  
    
  
  Publications

      </a>
    </li>
  

      
        
  
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
  
    
  
  Software

      </a>
    </li>
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../teaching/teaching/" class="md-tabs__link">
          
  
  
  Teaching

        </a>
      </li>
    
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../people/" class="md-tabs__link">
        
  
  
    
  
  People

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../careers/" class="md-tabs__link">
        
  
  
    
  
  Careers

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../contact/" class="md-tabs__link">
        
  
  
    
  
  Contact

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


  

<nav class="md-nav md-nav--primary md-nav--lifted md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Robinson Lab" class="md-nav__button md-logo" aria-label="Robinson Lab" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    Robinson Lab
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../publications/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Publications
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Software
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Software
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#the-human-phenotype-ontology-hpo" class="md-nav__link">
    <span class="md-ellipsis">
      The Human Phenotype Ontology (HPO)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#the-medical-action-ontology-maxo" class="md-nav__link">
    <span class="md-ellipsis">
      The Medical Action Ontology (MAxO)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#the-global-alliance-for-health-ga4gh-phenopacket-schema" class="md-nav__link">
    <span class="md-ellipsis">
      The Global Alliance for Health (GA4GH) Phenopacket Schema
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#exomiser" class="md-nav__link">
    <span class="md-ellipsis">
      Exomiser
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phenomizer" class="md-nav__link">
    <span class="md-ellipsis">
      Phenomizer
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#phenol" class="md-nav__link">
    <span class="md-ellipsis">
      Phenol
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lirical" class="md-nav__link">
    <span class="md-ellipsis">
      LIRICAL
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#squirls" class="md-nav__link">
    <span class="md-ellipsis">
      SQUIRLS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#svanna" class="md-nav__link">
    <span class="md-ellipsis">
      SvAnna
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#chip-seq-software" class="md-nav__link">
    <span class="md-ellipsis">
      ChIP-seq software
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gopher" class="md-nav__link">
    <span class="md-ellipsis">
      GOPHER
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#hba-deals" class="md-nav__link">
    <span class="md-ellipsis">
      HBA-DEALS
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#isopret" class="md-nav__link">
    <span class="md-ellipsis">
      isopret
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Teaching
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Teaching
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../teaching/teaching/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Intro
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../teaching/genomic-algorithms/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Genomic algorithms
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../teaching/genetik/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Genetik für Bioformatiker
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../people/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    People
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../careers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Careers
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../contact/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Contact
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



<h1 id="software">Software</h1>
<p>The Robinson lab develops a wide range of algorithms, computational resources, and applications. We use Java, Python, R, and occasionally
C++, Perl, or Rust. We emphasize good software engineering practices in bioinformatics. This page provides an overview of some of the
software or computational resources of the lab.</p>
<h2 id="the-human-phenotype-ontology-hpo">The Human Phenotype Ontology (HPO)</h2>
<p>The <a href="http://www.human-phenotype-ontology.org">Human Phenotype Ontology (HPO)</a> aims to provide a standardized vocabulary of phenotypic abnormalities encountered in human disease.
Terms in the HPO describe individual phenotypic abnormalities such as atrial septal defect.
The HPO is developed as a part of the <a href="http://monarchinitiative.org">Monarch Initiative</a>.</p>
<p><a class="md-button md-button--primary" href="https://github.com/obophenotype/human-phenotype-ontology">HPO :material-github:</a></p>
<figure>
<p><img alt="HPO" src="../img/hpo-kpmp.jpg" width="400" /></p>
<figcaption>Using HPO to suppor the Kidney Precision Medicine Project (KPMP). Figure from 
Köhler et al (2021) The Human Phenotype Ontology in 2021. Nucleic Acids Res; 49:D1207-D1217
</figcaption>
</figure>
<h2 id="the-medical-action-ontology-maxo">The Medical Action Ontology (MAxO)</h2>
<p>the Medical Action Ontology (MAxO), the first ontology specifically designed to organize medical procedures, therapies, and interventions in a structured way. 
Currently, MAxO contains 1757 medical action terms added through a combination of manual and semi-automated processes.</p>
<p><a class="md-button md-button--primary" href="https://github.com/monarch-initiative/maxo">MAxO :material-github:</a></p>
<h2 id="the-global-alliance-for-health-ga4gh-phenopacket-schema">The Global Alliance for Health (GA4GH) Phenopacket Schema</h2>
<p>The Phenopacket Schema represents an open standard for sharing disease and phenotype information to improve our ability to understand, diagnose, and treat both rare and common diseases. A Phenopacket links detailed phenotype descriptions with disease, patient, and genetic information, enabling clinicians, biologists, and disease and drug researchers to build more complete models of disease. The standard is designed to encourage wide adoption and synergy between the people, organizations and systems that comprise the joint effort to address human disease and biological understanding.</p>
<figure>
<p><img alt="GA4GH Phenopacket Schema" src="../img/phenopacket-summary.png" width="300" /></p>
<figcaption>GA4GH Phenopacket Schema
</figcaption>
</figure>
<ul>
<li><a href="https://pubmed.ncbi.nlm.nih.gov/35705716/">Jacobsen JOB, et al. (2022) The GA4GH Phenopacket schema defines a computable representation of clinical data. Nat Biotechnol. 40:817-820</a></li>
</ul>
<p><a class="md-button md-button--primary" href="https://github.com/phenopackets/phenopacket-schema">Phenopacket-Schema :material-github:</a></p>
<ul>
<li><a href="https://pubmed.ncbi.nlm.nih.gov/37196000/">Danis D,et al. (2023) Phenopacket-tools: Building and validating GA4GH Phenopackets. PLoS One. 18:e0285433</a>;</li>
</ul>
<p><a class="md-button md-button--primary" href="https://github.com/phenopackets/phenopacket-tools">Phenopacket TOOLS :material-github:</a></p>
<h2 id="exomiser">Exomiser</h2>
<p>The Exomiser is a Java program that functionally annotates and prioritises variants from whole-exome sequencing data starting from a 
VCF file. The Exomiser was developed by our group, Damian Smedley and Jules Jacobsen of the Mouse Informatics Group at the Sanger Institute (now at the 100,000 Genomes Project and Quenn Mary's University in London), and other members of the <a href="http://monarchinitiative.org">Monarch Initiative</a>.</p>
<ul>
<li><a href="https://pubmed.ncbi.nlm.nih.gov/24162188/" target="_blank">Robinson et al., 2014, Genome Research</a> and <a href="http://www.ncbi.nlm.nih.gov/pubmed/26562621" target="_blank">Smedley et al. (2015)</a></li>
</ul>
<p><a class="md-button md-button--primary" href="https://github.com/exomiser/Exomiser">Exomiser :material-github:</a></p>
<figure>
<p><img alt="Exomiser" src="../img/exomiser-alg.png" width="400" /></p>
<figcaption>Overview of Exomiser algorithm
</figcaption>
</figure>
<h2 id="phenomizer">Phenomizer</h2>
<p>The <a href="http://compbio.charite.de/phenomizer">Phenomizer</a>  aims to help clinicians to identify the correct differential diagnosis 
in the field of human genetics.
The user enters the signs/symptoms of the patient encoded as terms from the 
<a href="http://www.human-phenotype-ontology.org">HPO</a>. 
The software then ranks all diseases from OMIM, Orphanet, and DECIPHER by a score that reflects how well the phenotypic profiles of the patient and the disease match to each other.</p>
<ul>
<li>See also <a href="http://www.ncbi.nlm.nih.gov/pubmed/19800049">Kohler, S. et al., 2009</a></li>
</ul>
<figure>
<p><img alt="Exomiser" src="../img/LDSphenomizer1.png" width="400" /></p>
<figcaption>Phenomizer Graphical User Interface
</figcaption>
</figure>
<h2 id="phenol">Phenol</h2>
<p>Phenol is a Java library for working with JSON phenotype ontologies including especially the Human Phenotype Ontology and 
the Mammalian Phenotype Ontology and associate phenotype annotation files.</p>
<p><a class="md-button md-button--primary" href="https://github.com/monarch-initiative/phenol">Phenol :material-github:</a></p>
<h2 id="lirical">LIRICAL</h2>
<p>LIkelihood Ratio Interpretation of Clinical AbnormaLities (LIRICAL)   exploits the likelihood ratio (LR) 
framework to provide an estimate of (1) the posttest probability of candidate diagnoses, 
(2) the LR for each observed HPO phenotype, and (3) the predicted pathogenicity of observed genotypes.</p>
<p><a class="md-button md-button--primary" href="https://github.com/TheJacksonLaboratory/LIRICAL">LIRICAL :material-github:</a></p>
<figure>
<p><img alt="GA4GH Phenopacket Schema" src="../img/lirical.jpg" width="400" /></p>
<figcaption>Robinson PN, et al. (2020) Interpretable Clinical Genomics with a Likelihood Ratio Paradigm. Am J Hum Genet; 107:403-417.
</figcaption>
</figure>
<h2 id="squirls">SQUIRLS</h2>
<p>The Super Quick Information-content Random-forest Learning of Splice variants (SQUIRLS) algorithm 
generates a small set of interpretable features for machine learning by calculating the information-content 
of wild-type and variant sequences of canonical and cryptic splice sites, assessing changes in candidate 
splicing regulatory sequences, and incorporating characteristics of the sequence such as exon length, 
disruptions of the AG exclusion zone, and conservation. </p>
<p><a class="md-button md-button--primary" href="https://github.com/TheJacksonLaboratory/SQUIRLS">SQUIRLS :material-github:</a></p>
<figure>
<p><img alt="SQUIRLS" src="../img/squirls.jpg" width="400" /></p>
<figcaption>Danis D, et al (2021) Interpretable prioritization of splice variants in diagnostic next-generation sequencing. 
Am J Hum Genet; 108:1564-1577.
</figcaption>
</figure>
<h2 id="svanna">SvAnna</h2>
<p>Structural variants (SVs) are implicated in the etiology of Mendelian diseases but have been systematically 
underascertained owing to sequencing technology limitations. Long-read sequencing enables comprehensive 
detection of SVs, but approaches for prioritization of candidate SVs are needed. Structural variant 
Annotation and analysis (SvAnna) assesses all classes of SVs and their intersection with transcripts and regulatory 
sequences, relating predicted effects on gene function with clinical phenotype data. </p>
<p><a class="md-button md-button--primary" href="https://github.com/TheJacksonLaboratory/SvAnna">SvAnna :material-github:</a></p>
<figure>
<p><img alt="SvAnna" src="../img/svanna.png" width="400" /></p>
<figcaption>Danis D. et al. (2022) SvAnna: efficient and accurate pathogenicity prediction of coding and regulatory structural variants in long-read genome sequencing. 
Genome Med; 14:44. 
</figcaption>
</figure>
<h2 id="chip-seq-software">ChIP-seq software</h2>
<p>Chromatin immunoprecipitation coupled with next-generation sequencing (ChIP-seq) is a powerful technology to identify the genome-wide locations of transcription factors and other DNA binding proteins. Computational ChIP-seq peak calling infers the location of protein-DNA interactions based on various measures of enrichment of sequence reads.
Our algorithm, Q,  uses an assessment of the quadratic enrichment of reads to center candidate peaks followed by statistical analysis of saturation of candidate peaks by 5' ends of reads. We show that our method not only is substantially faster than several competing methods but also demonstrates statistically significant advantages with respect to reproducibility of results and in its ability to identify peaks with reproducible binding site motifs.</p>
<p><a class="md-button md-button--primary" href="https://github.com/charite/Q">Q :material-github:</a></p>
<figure>
<p><img alt="Q algorithm" src="../img/Q-algorithm.jpg" width="400" /></p>
<figcaption>Hansen P, et al. (2015) Saturation
analysis of ChIP-seq data for reproducible identification of binding peaks. Genome Res 25:1391-400.
</figcaption>
</figure>
<h2 id="gopher">GOPHER</h2>
<p>Target enrichment combined with chromosome conformation capturing methodologies such as capture Hi-C (CHC) can be used 
 to investigate spatial layouts of genomic regions with high resolution and at scalable costs. A common application of CHC is the investigation of regulatory elements that are in contact with promoters, but CHC can be used for a range of other applications. 
GOPHER (Generator Of Probes for capture Hi-C Experiments at high Resolution) is an easy-to-use and robust desktop application for CHC probe design</p>
<p><a class="md-button md-button--primary" href="https://github.com/TheJacksonLaboratory/Gopher">GOPHER :material-github:</a></p>
<figure>
<p><img alt="HBA-DEALS" src="../img/gopher.png" width="400" /></p>
<figcaption>Hansen P et al. (2019) GOPHER: Generator Of Probes for capture Hi-C Experiments at high Resolution. BMC Genomics; 20(1):40.</figcaption>
</figure>
<h2 id="hba-deals">HBA-DEALS</h2>
<p>Hierarchical Bayesian Analysis of Differential Expression and ALternative Splicing (HBA-DEALS) is an R package that 
simultaneously characterizes differential gene expression and alternative splicing in high-throughput gene expression data. 
It uses counts data for isoforms (alternative transcripts of a gene) to infer the parameters of a hierarchical 
Bayesian model of expression and splicing. It then uses the posterior of the parameters to determine the existence 
of differential expression and/or differential alternative splicing. Isoform counts can be derived from short-read 
sequencing RNA-Seq data or from long-read RNA sequencing data such as that generated by the PacBio SequelII platform.</p>
<p><a class="md-button md-button--primary" href="https://github.com/TheJacksonLaboratory/HBA-DEALS">HBA-DEALS :material-github:</a></p>
<figure>
<p><img alt="HBA-DEALS" src="../img/hbadeals.png" width="400" /></p>
<figcaption>Karlebach G, et al. (2020) HBA-DEALS: accurate and simultaneous identification of differential expression and splicing using hierarchical Bayesian analysis. Genome Biol;21:171. 
</figcaption>
</figure>
<h2 id="isopret">isopret</h2>
<p>isopret (<em>isoform interpretation</em>) is a method that uses expectation–maximization to infer isoform-specific 
functions based on the relationship between sequence and functional isoform similarity. 
We predicted isoform-specific functional annotations for 85 617 isoforms of 17 900 protein-coding human genes spanning 
a range of 17 430 distinct gene ontology terms. </p>
<p><a class="md-button md-button--primary" href="https://github.com/TheJacksonLaboratory/isopretEM">isopretEM :material-github:</a></p>
<figure>
<p><img alt="HBA-DEALS" src="../img/isopret.jpg" width="400" /></p>
<figcaption>Karlebach G (2023) An expectation-maximization framework for comprehensive prediction of isoform-specific functions. Bioinformatics; 39(4):btad132.</figcaption>
</figure>







  
  






                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      &copy; 2023 <a href="https://github.com/pnrobinson"  target="_blank" rel="noopener">Peter N Robinson</a>

    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        
<div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://github.com/pnrobinson" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M202.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1M496 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2m-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3m-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://twitter.com/pnrobins" target="_blank" rel="noopener" title="twitter.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M459.4 151.7c.3 4.5.3 9.1.3 13.6 0 138.7-105.6 298.6-298.6 298.6-59.5 0-114.7-17.2-161.1-47.1 8.4 1 16.6 1.3 25.3 1.3 49.1 0 94.2-16.6 130.3-44.8-46.1-1-84.8-31.2-98.1-72.8 6.5 1 13 1.6 19.8 1.6 9.4 0 18.8-1.3 27.6-3.6-48.1-9.7-84.1-52-84.1-103v-1.3c14 7.8 30.2 12.7 47.4 13.3-28.3-18.8-46.8-51-46.8-87.4 0-19.5 5.2-37.4 14.3-53C87.4 130.8 165 172.4 252.1 176.9c-1.6-7.8-2.6-15.9-2.6-24C249.5 95.1 296.3 48 354.4 48c30.2 0 57.5 12.7 76.7 33.1 23.7-4.5 46.5-13.3 66.6-25.3-7.8 24.4-24.4 44.8-46.1 57.8 21.1-2.3 41.6-8.1 60.4-16.2-14.3 20.8-32.2 39.3-52.6 54.3"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://www.linkedin.com/in/peter-n-robinson-b7833811/" target="_blank" rel="noopener" title="www.linkedin.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path fill="currentColor" d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3M135.4 416H69V202.2h66.5V416zM102.2 96a38.5 38.5 0 1 1 0 77 38.5 38.5 0 1 1 0-77m282.1 320h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.sections", "toc.integrate", "navigation.top", "search.suggest", "search.highlight", "content.tabs.link", "content.code.annotation", "content.code.copy"], "search": "../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.92b07e13.min.js"></script>
      
        <script src="../assets/javascripts/badge.js" async></script>
      
    
  </body>
</html>