
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://robinsongroup.github.io/people/">
      
      
        <link rel="prev" href="../teaching/genetik/">
      
      
        <link rel="next" href="../careers/">
      
      
        
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.3">
    
    
      
        <title>People - Robinson Lab</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.484c7ddc.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.ab4e12ef.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/badge.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  
<meta property="og:type" content="website" />
<meta property="og:title" content="People - Robinson Lab" />
<meta property="og:image" content="https://robinsongroup.github.io/assets/images/social/people.png" />
<meta property="og:image:type" content="image/png" />
<meta property="og:image:width" content="1200" />
<meta property="og:image:height" content="630" />
<meta property="og:url" content="https://robinsongroup.github.io/people/" />
<meta property="twitter:card" content="summary_large_image" />
<meta property="twitter:title" content="People - Robinson Lab" />
<meta property="twitter:image" content="https://robinsongroup.github.io/assets/images/social/people.png" />
</head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="teal" data-md-color-accent="purple">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#people" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Robinson Lab" class="md-header__button md-logo" aria-label="Robinson Lab" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Robinson Lab
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              People
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="teal" data-md-color-accent="purple"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6m0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4M7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="teal" data-md-color-accent="lime"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../publications/" class="md-tabs__link">
        
  
  
    
  
  Publications

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../software/" class="md-tabs__link">
        
  
  
    
  
  Software

      </a>
    </li>
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../teaching/teaching/" class="md-tabs__link">
          
  
  
  Teaching

        </a>
      </li>
    
  

      
        
  
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
  
    
  
  People

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../careers/" class="md-tabs__link">
        
  
  
    
  
  Careers

      </a>
    </li>
  

      
        
  
  
  
  
    <li class="md-tabs__item">
      <a href="../contact/" class="md-tabs__link">
        
  
  
    
  
  Contact

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


  

<nav class="md-nav md-nav--primary md-nav--lifted md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Robinson Lab" class="md-nav__button md-logo" aria-label="Robinson Lab" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    Robinson Lab
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Home
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../publications/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Publications
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../software/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Software
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Teaching
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Teaching
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../teaching/teaching/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Intro
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../teaching/genomic-algorithms/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Genomic algorithms
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../teaching/genetik/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Genetik für Bioformatiker
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    
  
    People
  

    
  </span>
  
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    
  
    People
  

    
  </span>
  
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#peter-n-robinson-md" class="md-nav__link">
    <span class="md-ellipsis">
      
        Peter N. Robinson, MD
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lab-members" class="md-nav__link">
    <span class="md-ellipsis">
      
        Lab Members
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Lab Members">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#martha-beckwith-phd" class="md-nav__link">
    <span class="md-ellipsis">
      
        Martha Beckwith, Ph.D.
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ben-coleman" class="md-nav__link">
    <span class="md-ellipsis">
      
        Ben Coleman
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#daniel-danis-phd" class="md-nav__link">
    <span class="md-ellipsis">
      
        Daniel Danis, Ph.D.
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#michael-gargano" class="md-nav__link">
    <span class="md-ellipsis">
      
        Michael Gargano
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#peter-hansen-phd" class="md-nav__link">
    <span class="md-ellipsis">
      
        Peter Hansen, Ph.D.
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#kristin-kohler" class="md-nav__link">
    <span class="md-ellipsis">
      
        Kristin Köhler
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lukas-ramlow-phd" class="md-nav__link">
    <span class="md-ellipsis">
      
        Lukas Ramlow, Ph.D.
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rouven-reuter" class="md-nav__link">
    <span class="md-ellipsis">
      
        Rouven Reuter
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#leonardo-chimirri-phd" class="md-nav__link">
    <span class="md-ellipsis">
      
        Leonardo Chimirri, Ph.D.
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#enock-niyonkuru" class="md-nav__link">
    <span class="md-ellipsis">
      
        Enock Niyonkuru
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#lauren-rekerle" class="md-nav__link">
    <span class="md-ellipsis">
      
        Lauren Rekerle
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#patrick-nairne-phd" class="md-nav__link">
    <span class="md-ellipsis">
      
        Patrick Nairne, Ph.D.
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#varenya-jain" class="md-nav__link">
    <span class="md-ellipsis">
      
        Varenya Jain
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#wiebke-hartung" class="md-nav__link">
    <span class="md-ellipsis">
      
        Wiebke Hartung
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#alumni" class="md-nav__link">
    <span class="md-ellipsis">
      
        Alumni
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../careers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Careers
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../contact/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Contact
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  



<h1 id="people">People</h1>
<h2 id="peter-n-robinson-md">Peter N. Robinson, MD</h2>
<p>We are living in a time of unparalleled opportunity to extend our understanding of human disease and improve the
care of patients with precision genomic medicine. Sophisticated bioinformatics and computational biology are
essential to achieve the full potential of genomics for science as well as for patients. Peter Robinson,
Professor and Donald A. Roux Chair, Genomics and Computational Biology, leads a research group dedicated to the
development of algorithms and computational resources for genomics. Highlights
of the lab’s work include the <a href="http://www.human-phenotype-ontology.org">Human Phenotype Ontology</a>,
the <a href="https://monarchinitiative.org/page/exomes">Exomiser</a> suite  of tools for exome and genome analysis,
and algorithms for ChIP-seq and immunogenomics. Peter studied Mathematics (Bachelor) at Columbia College,
Medicine at the University of Pennsylvania, and Computer Science (Master's) at Columbia University.
He completed an internship in Primary Care Internal Medicine at Yale University, and a residency (Facharzt) in
Pediatrics  and obtained a Habilitation (roughly equivalent
to a PhD) in Human Genetics at the Charité - Universitätsmedizin Berlin. He led the
Bioinformatics group at the Institute for Medical Genetics and Human Genetics at the Charité from 2004-2016.
From 2016-2023, he led a research group at  the Jackson Laboratory (JAX) for Genomic Medicine in Connecticut, USA.
From 2024, his primary affiliation will be with the Berlin Institute of Health (BIH) and he will continue to
be affiliated with JAX.</p>
<figure>
<p><img alt="Peter Robinson" src="../img/people/peter-robinson.jpg" width="200" /></p>
<figcaption>Peter Robinson
</figcaption>
</figure>
<h2 id="lab-members">Lab Members</h2>
<h3 id="martha-beckwith-phd">Martha Beckwith, Ph.D.</h3>
<figure>
<p><img alt="Martha Beckwith" src="../img/people/mbeckwith.png" width="200" /></p>
<figcaption>Martha Beckwith
</figcaption>
</figure>
<p>Martha Beckwith obtained her PhD in Physical Chemistry from Cornell
University. During her PhD, she also conducted research at the Max-Planck-
Institute for Chemical Energy Conversion in Germany. Prior to joining the
Robinson Lab, she was a Postdoctoral Research Associate at Lawrence
Livermore National Laboratory, and then a Computational Scientist at the
CUNY Advanced Science Research Center. Her current work focuses on
developing software for incorporating clinical intuition in phenotype-driven
prioritization of rare diseases.</p>
<h3 id="ben-coleman">Ben Coleman</h3>
<figure>
<p><img alt="Leigh Carmody" src="../img/people/ben-coleman.png" width="200" /></p>
<figcaption>Ben Coleman
</figcaption>
</figure>
<p>I am a current MD-PhD student at UConn School of Medicine with a background in physics.
I joined the Robinson Lab after two years of medical school to conduct my thesis research.
My research focuses on using the Human Phenotype Ontology (HPO) to describe and understand
the phenotypic features of neurodevelopmental disorders in a generalizable and computable format.
By improving the way we describe patient phenotypes, we will improve our ability to identify
the genetic drivers of these phenotypes. To do this, I work with domain experts to expand the
HPO's terminology for neurodevelopmental disorders and to develop tools to translate clinical
measurements to the HPO.</p>
<h3 id="daniel-danis-phd">Daniel Danis, Ph.D.</h3>
<figure>
<p><img alt="Daniel Danis" src="../img/people/daniel-danis.jpg" width="200" /></p>
<figcaption>Daniel Danis
</figcaption>
</figure>
<p>Daniel has a M.Sc. in Pharmacy from the Comenius University, Bratislava, Slovakia. He is currently working on his Ph.D., mainly focused on the molecular basis and pathomechanisms of rare hereditary diseases in human patients. He has experience in the assembly of custom UNIX-based bioinformatics pipelines for whole exome sequencing data analysis that have identified disease-causing variants in several Slovak families.
His work in the Robinson lab involves the development of algorithms for prioritizing exome and genome variants. These new algorithms are designed to integrate into the Exomiser and Genomiser frameworks. In addition, he works on tools for the biocuration of published disease-causing variants.</p>
<h3 id="michael-gargano">Michael Gargano</h3>
<figure>
<p><img alt="Michael Gargano" src="../img/people/michael-g.png" width="200" /></p>
<figcaption>Michael Gargano
</figcaption>
</figure>
<p>Michael Gargano has an M.S in Bioinformatics from Northeastern University. First starting as a UI
Developer at Cigna, he joined both The Jackson Laboratory &amp; the Robinson Lab in 2017, and has grown to
a Senior Scientific Software Engineer. Bringing the Human Phenotype Ontology to the modern web was his
flagship project. He now specializes in data engineering on the cloud, restful applications, bioinformatics
pipelines, and architecting software solutions for translational &amp; clinical genomics.</p>
<h3 id="peter-hansen-phd">Peter Hansen, Ph.D.</h3>
<figure>
<p><img alt="Peter Hansen" src="../img/people/peter-hansen.jpg" width="200" /></p>
<figcaption>Peter Hansen
</figcaption>
</figure>
<p>Peter Hansen studied bioinformatics at the Free University of Berlin from 2003 to 2010.
In 2011, he began his professional career at the Institute for Medical Genetics and Human Genetics at the Charité hospital in Berlin,
where he set up analysis pipelines for high-throughput sequencing data.
By analyzing ChIP-seq data from different collaborators,
he helped to elucidate the roles of various transcription factors in different contexts,
for example HOXD13 and PITX1 in limb development, PAX5 in classical Hodgkin's lymphoma, or E2F6 in CGI methylation.
In 2014, he continued his education as a doctoral student.
He developed the ChIP-seq peak calling software
<a href="https://github.com/charite/Q">Q and Q-nexus</a>.
Moreover, he led the project "Genomic diagnostics for the regulatory genome" at the Charité on behalf of Peter Robinson,
with a focus on conducting capture Hi-C experiments and data analysis.
In 2019, he received his
<a href="https://refubium.fu-berlin.de/bitstream/handle/fub188/24639/thesis_Hansen.pdf">Ph.D. in Bioinformatics</a>
from the Department of Mathematics and Computer Science at the Free University Berlin and joined The Jackson Laboratory
for Genomic Medicine as a Bioinformatics Analyst.
He developed <a href="https://github.com/TheJacksonLaboratory/Gopher">GOPHER</a>,
a desktop application for capture Hi-C probe design,
and <a href="https://github.com/TheJacksonLaboratory/diachromatic">Diachromatic</a>,
a toolset for preprocessing and quality control of Hi-C and capture Hi-C data.
Furthermore, he characterized a feature of Hi-C data and used it to assess and mitigate technical bias.
In parallel, he contributed to team projects on a variety of topics
including the analysis of differential expression and splicing,
prediction of isoform functions,
prediction of associations between protein kinases and cancer,
and challenges in biomedical knowledge graph representation.
More recently, he has been working on ontologizing experimental mouse data from the Mouse Phenome Database at JAX and
representing it as Phenopackets.</p>
<h3 id="kristin-kohler">Kristin Köhler</h3>
<figure>
<p><img alt="Kristin Köhler" src="../img/people/KristinKoehler.jpg" width="200" /></p>
<figcaption>Kristin Köhler
</figcaption>
</figure>
<p>Kristin earned her B.Sc. and M.Sc. in Bioinformatics from Free Universität Berlin. After completing her bachelor’s studies, she joined the Kircher lab at the Berlin Institute of Health (BIH), where she developed methods to analyze cfDNA from liquid biopsies and built predictive models for cancer diagnosis. During her master’s studies, she joined the Conrad lab at BIH, where she developed data processing pipelines for single-cell transcriptomics using nanopore technologies. This work enabled detailed analysis of transcript expression and splicing at the single-cell level. In May 2024, Kristin began her PhD at BIH, working jointly in the groups of Peter N. Robinson and Christian Conrad. Bridging the gap between wet and dry lab research, she focuses on developing software to analyze large-scale transcriptomic data, leveraging cutting-edge single-cell and spatial technologies.</p>
<h3 id="lukas-ramlow-phd">Lukas Ramlow, Ph.D.</h3>
<figure>
<p><img alt="Lukas Ramlow" src="../img/people/Lukas-Ramlow.jpg" width="200" /></p>
<figcaption>Lukas Ramlow
</figcaption>
</figure>
<p>Lukas joined the Robinson Lab in May 2025 as a Postdoctoral Researcher, transitioning from theoretical physics to medical informatics. He holds both a B.Sc. and M.Sc. in Physics from the Technical University of Berlin.</p>
<p>He earned his PhD in Statistical Physics from the Humboldt University of Berlin and the Bernstein Center for Computational Neuroscience. In his doctoral research, he developed mathematical models to investigate the cellular physiology underlying non-renewal spiking. He analyzed how these cellular processes influence the spiking statistics, and consequently the information transmission, of complex neural and calcium signaling systems.</p>
<p>Motivated by his interest in the intersection of biology and advanced statistical mechanics, Lukas uses rigorous theoretical methods to solve complex problems in translational bioinformatics. His goal is to extract meaningful insights from large-scale biomedical data and bridge the gap between theory and clinical application.</p>
<p><a href="https://scholar.google.com/citations?user=6zqlhhoAAAAJ&amp;hl=de">Google Scholar</a></p>
<p><a href="https://orcid.org/0000-0001-6089-0613">ORCID</a></p>
<h3 id="rouven-reuter">Rouven Reuter</h3>
<figure>
<p><img alt="Rouven Reuter" src="../img/people/Rouven-Reuter.jpg" width="200" /></p>
<figcaption>Rouven Reuter
</figcaption>
</figure>
<p>Rouven graduated in Media Technology (B.Eng.) at the Hochschule Emden/Leer and Media Informatics (M.Sc.) at the Berliner
Hochschule für Technik. His master's thesis was done in cooperation with the nephrology department of the Charité
Berlin, where he trained a deep neural network model using tabular and unstructured text data of digital health records
to predict acute kidney injury. For the following four years, he gathered experience as a Machine Learning Engineer in
the industry and started his PhD in June 2025 by joining the Robinson Lab at BIH. The medical sector was a natural
choice for him, as he is fueled by a passion to build stuff that actually matters.
<a href="https://www.smart-monkey.blog/">Blog</a></p>
<h3 id="leonardo-chimirri-phd">Leonardo Chimirri, Ph.D.</h3>
<figure>
<p><img alt="Leonardo Chimirri" src="../img/people/Chimirri_Leonardo_8x10.jpg" width="200" /></p>
<figcaption>Leonardo Chimirri
</figcaption>
</figure>
<p>Leonardo received his B.Sc. in Physics and M.Sc. in Theoretical Physics (2013-2018) at University of Torino, Italy, before moving to Berlin where he earned his Ph.D. in Theoretical Particle Physics (2019-2023) at Humboldt University Berlin and DESY-Zeuthen. During his Master and then Ph.D. theses he focused on lattice field theories, a computationally intensive approach based on stochastic simulations. There he expanded his interest and experience in programming, mathematical modelling, data analysis and machine learning. Driven by his deep desire to apply his skills to improve people's lives and health and given his broad scientific interest, among which computational biology and genetics stand out, he decided to change career focus and joined the Robinson Lab in April 2024. This was facilitated by coming into contact with knowledge graphs, ontologies, and rare genetic diseases when working as a volunteer at MI4People gGmbH (German non-profit), which he still supports whenever possible. He has been focusing on LLMs for differentials diagnostics of rare diseases throughout his first period in the Lab. Other topics he is starting to explore are bayesian networks for differential diagnostics and training of machine learning models for prioritizing pathogenic variants of non-coding DNA regions.
<a href="https://scholar.google.com/citations?user=9ltLmKMAAAAJ&amp;hl=en">Google Scholar</a>, <a href="https://orcid.org/0000-0002-6912-8518">ORCID</a></p>
<h3 id="enock-niyonkuru">Enock Niyonkuru</h3>
<figure>
<p><img alt="Enock Niyonkuru" src="../img/people/enock.png" width="200" /></p>
<figcaption>Enock Niyonkuru
</figcaption>
</figure>
<p>Enock is pursuing his BS in Computer Science at Trinity College, CT.
He became a member of The Robinson Lab in Fall 2021. From a young age, Enock was deeply intrigued by the
potential of Artificial Intelligence in enhancing healthcare. This interest was further cemented
during his high school years when he shadowed physicians at a Rwandan hospital and recognized the
pressing need for advanced technology in the medical field. Within the Lab, Enock focuses on developing
software and algorithms to refine text mining techniques. He integrates Machine Learning and
Large Language Models to assist in the curation of the Medical Action Ontology (MAxO) and contributes to
various other bioinformatics initiatives.</p>
<h3 id="lauren-rekerle">Lauren Rekerle</h3>
<figure>
<p><img alt="Lauren Rekerle" src="../img/people/LaurenRekerle.jpg" width="200" /></p>
<figcaption>Lauren Rekerle
</figcaption>
</figure>
<p>Lauren Rekerle received her B.Sc. in Bioinformatics from Michigan Technological University (2017) and her M.Sc. in Computational Biology from the University of Minnesota (2019). During her time as a master’s student, she worked with the National Marrow Donor Program’s bioinformatics team. After she graduated, she spent over 2 years working for the Genomics Unit in the Mayo Clinic’s IT department. In spring of 2022, Lauren joined Peter Robinson’s lab as a bioinformatician. Since then, she has collected and analyzed data from predicted synthetic lethality interactions, designed and aided in the development of <a href="https://github.com/monarch-initiative/gpsea">GPSEA</a>, and used the software to find new genotype-phenotype correlations.</p>
<h3 id="patrick-nairne-phd">Patrick Nairne, Ph.D.</h3>
<figure>
<p><img alt="Patrick Nairne" src="../img/people/PatrickNairne.jpg" width="200" /></p>
<figcaption>Patrick Nairne
</figcaption>
</figure>
<p>Patrick Nairne received his PhD in Mathematics from the University of Oxford in 2024. His research primarily focused on coarse metric embeddings between hyperbolic metric spaces and products of trees. He also researched certain algorithmic properties of hyperbolic groups. Since joining the Robinson Lab in June 2025, he has been applying machine learning methods to estimate the pathogenicity of non-coding variants. He is also interested in generating Phenopackets (a way of structuring patient data that is conducive to machine learning and data analysis) from a variety of different medical data sources. His personal academic website can be found <a href="https://psnairne.github.io">here</a>.</p>
<h3 id="varenya-jain">Varenya Jain</h3>
<figure>
<p><img alt="Varenya Jain" src="../img/people/Varenya-Jain.jpg" width="200" /></p>
<figcaption>Varenya Jain
</figcaption>
</figure>
<p>Varenya Jain earned his B.Sc. in Integrative Biology with a minor in Computational Science and Engineering from the University of Illinois Urbana-Champaign in 2024. In August 2024, he began collaborating with Peter Robinson, Ronald Wapner, and Michael Duyzend as part of the Fetal Anomalies Repository Leadership Team, a joint initiative between Columbia University's Division of Women's Genetics and the Broad Institute of MIT and Harvard, to develop a genotype-phenotype database for rare diseases within the Fetal Sequencing Consortium. His current work involves training local large language models to extract Human Phenotype Ontology terms from prenatal ultrasound data and creating federated phenopacket repositories.</p>
<p>Previously, Varenya conducted research in computational phylogenetics, developing a novel method for assembling phylogenetic markers from low-coverage whole-genome sequence data using zebrafish. He also worked as a bioinformatician, integrating DRAGEN enrichment-based CNV detection into clinical variant interpretation workflows. His broader research background spans evolutionary biology, multiomics, and high-performance computing.</p>
<h3 id="wiebke-hartung">Wiebke Hartung</h3>
<figure>
<p><img alt="Wiebke Hartung" src="../img/people/Wiebke-Hartung.jpg" width="200" /></p>
<figcaption>Wiebke Hartung
</figcaption>
</figure>
<p>Wiebke Hartung is an experienced administrative and operations professional currently serving as Office Managerin at Charite - Universitatsmedizin Berlin, where she works in the research groups of the Berlin Institute of Health for Prof. Dr. Peter Robinson and Prof. Dr. Sylvia Thun. Before taking on this role, she was part of the Clinical Study Center (CSC) for over five years, contributing to the Clinical Trial Unit (CTU) Site Management. She was also an elected member of the BIH staff council, representing employees' interests and advising on workplace matters. Her background blends expertise in human resources, budgeting, and operational coordination within complex medical research environments, making her a key link between scientific leadership, administrative structures, and institutional governance.</p>
<h3 id="alumni">Alumni</h3>
<table>
<thead>
<tr>
<th style="text-align: left;">Almunus/a</th>
<th style="text-align: left;">Name</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Layal Abo-Kayal. PhD</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Miriam Sandya Bauer, Dr. med.</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Dr. Sebastian Bauer, PhD. (<a href="https://www.htw-berlin.de/hochschule/personen/person/?eid=8385">HTW Berlin</a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/hannah-blau.jpg" width="40" /></td>
<td style="text-align: left;">Hannah Blau, Ph.D</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Patrick Booms, Ph.D.</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/leigh-carmody.jpg" width="30" /></td>
<td style="text-align: left;">Leigh Carmody, Ph.D.</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Manuel Holgrewe, Ph.D. (<a href="https://www.cubi.bihealth.org/team/manuel-holtgrewe/">CUBI/BIH</a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Verena Heinrich, Ph.D. (<a href="https://pure.mpg.de/cone/persons/resource/persons179829">MPG</a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Marten Jäger, PhD (<a href="https://genomics.charite.de/metas/person/person/address_detail/dr_marten_jaeger/">BIH Core Facility Genomik </a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/guy-karlebach.jpg" width="40" /></td>
<td style="text-align: left;">Guy Karlebach, Ph.D.</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Sebastian Köhler, Dr. rer. nat. (<a href="http://drseb.github.io" target="_blank">Homepage</a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Peter Krawitz, PhD. (<a href="https://www.igsb.uni-bonn.de/en">Uni Bonn</a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Leon Kuchenbecker, PhD (<a href="https://www.ghga.de/about-us/team-members/kuchenbecker-leon">GHGA</a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Martin-Atta Mensah, Dr. med.</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Begoña Muñoz-Garcia, PhD</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Dmitri Parkhomchuk, PhD</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/vida-ravanmehr.jpg" width="40" /></td>
<td style="text-align: left;">Vida Ravanmehr, Ph.D</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Christian Rödelsperger.PhD</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Max Schubach, PhD (<a href="https://kircherlab.github.io/people/max.html">Computational Genome Biology, BIH</a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Marcel H Schulz, Ph.D. (<a href="https://schulzlab.github.io/MS.html">Uni Frankfurt</a>)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Robin Steinhaus, MSc (BIH)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Oliver Stolpe (BIH)</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/aaron-zhang.jpg" width="40" /></td>
<td style="text-align: left;">Xingmin “Aaron” Zhang, Ph.D.</td>
</tr>
<tr>
<td style="text-align: left;"><img align="left" alt="" src="../img/people/head-icon.png" width="30" /></td>
<td style="text-align: left;">Na Zhu. PhD</td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      &copy; 2023 <a href="https://github.com/pnrobinson"  target="_blank" rel="noopener">Peter N Robinson</a>

    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        
<div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://github.com/pnrobinson" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M202.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1M496 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2m-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3m-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://twitter.com/pnrobins" target="_blank" rel="noopener" title="twitter.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M459.4 151.7c.3 4.5.3 9.1.3 13.6 0 138.7-105.6 298.6-298.6 298.6-59.5 0-114.7-17.2-161.1-47.1 8.4 1 16.6 1.3 25.3 1.3 49.1 0 94.2-16.6 130.3-44.8-46.1-1-84.8-31.2-98.1-72.8 6.5 1 13 1.6 19.8 1.6 9.4 0 18.8-1.3 27.6-3.6-48.1-9.7-84.1-52-84.1-103v-1.3c14 7.8 30.2 12.7 47.4 13.3-28.3-18.8-46.8-51-46.8-87.4 0-19.5 5.2-37.4 14.3-53C87.4 130.8 165 172.4 252.1 176.9c-1.6-7.8-2.6-15.9-2.6-24C249.5 95.1 296.3 48 354.4 48c30.2 0 57.5 12.7 76.7 33.1 23.7-4.5 46.5-13.3 66.6-25.3-7.8 24.4-24.4 44.8-46.1 57.8 21.1-2.3 41.6-8.1 60.4-16.2-14.3 20.8-32.2 39.3-52.6 54.3"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://www.linkedin.com/in/peter-n-robinson-b7833811/" target="_blank" rel="noopener" title="www.linkedin.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3M135.4 416H69V202.2h66.5V416zM102.2 96a38.5 38.5 0 1 1 0 77 38.5 38.5 0 1 1 0-77m282.1 320h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "..", "features": ["navigation.tabs", "navigation.sections", "toc.integrate", "navigation.top", "search.suggest", "search.highlight", "content.tabs.link", "content.code.annotation", "content.code.copy"], "search": "../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.79ae519e.min.js"></script>
      
        <script src="../assets/javascripts/badge.js" async></script>
      
    
  </body>
</html>