# Careers

We welcome lab members of any nationality, sex, or gender, expect dedication and passion for bioinformatics, and provide training in 
academic research in bioinformatics including AI/ML, ontologies, algorithms, and software engineering.
Lab meetings and presentations are typically conducted in English, day to day conversations are conducted in German, English, or other languages, according to circumstances.

We are currently searching for motivated candidates for the following positions:

- [PhD position](https://karriere.charite.de/stellenangebote/detail/3994). Deadline for application: 07.02.2025 (Feb. 7th, 2025)
- [Postdoc positions (3)](https://karriere.charite.de/stellenangebote/detail/4190) Deadline for application: 07.02.2025 (Feb. 7th, 2025)



## Working in the Robinson lab

The [Robinson lab at the Berlin Institute of Health](https://www.bihealth.org/en/research/research-group/medical-computer-science-and-artificial-intelligence) is a highly interactive, supportive, and interdisciplinary group of faculty, staff scientists, postdoctoral fellows, and students and enjoy access to state-of-the-art computing environment at the BIH. 

The Robinson lab focuses on developing algorithms, ontologies, and software to advance research and support medical practice. The Robinson lab leads the development of the [Human Phenotype Ontology (HPO)](https://hpo.jax.org/), which has become the international standard for computational phenotype analysis in Mendelian disease and other areas. The lab has recently spearheaded the development of the [Medical Action Ontology (MAxO)](https://www.ebi.ac.uk/ols4/ontologies/maxo), which intends to provide a computational foundation for searching and analyzing treatments and other medical management with a focus on genetic disease. The lab develops software for phenotype (HPO) driven diagnostics including the [Phenomizer](https://pubmed.ncbi.nlm.nih.gov/19800049/) and [LIRICAL](https://pubmed.ncbi.nlm.nih.gov/32755546/). We develop graph algorithms including the first use of random-walk analysis for disease gene prediction, which was cited over 1400 times ([Walking the interactome...](https://pubmed.ncbi.nlm.nih.gov/18371930/)). We have developed machine-learning and statistical approaches towards predicting pathogenicity of variants in exome or genome sequencing, including [Genomiser](https://pubmed.ncbi.nlm.nih.gov/27569544/) and [SQUIRLS](https://pubmed.ncbi.nlm.nih.gov/34289339/). We develop software for the analysis of genomic data including [ChIP-seq-Q](https://pubmed.ncbi.nlm.nih.gov/26163319/), and [HBA-DEALS](https://pubmed.ncbi.nlm.nih.gov/32660516/), which uses a hierarchical Bayesian network to simultaneously assess gene expression and alternative splicing from RNA-seq data. Our [Exomiser](https://pubmed.ncbi.nlm.nih.gov/26562621/) software is widely used by projects such as the 100,000 Genomes project to assess diagnostic exome and genome data. 

Our lab strives to be a welcoming and supportive environment where people of all backgrounds have the opportunity to develop algorithms and software at the cutting edge of translational bioinformatics. Lab members typically focus on one or two areas of interest in fields such as ontologies, semantic algorithms, or machine learning. We provide extensive on-the-job training in algorithms and software engineering as needed for the projects. 

### Expectations

As a rule, we have learned about algorithms during the projects and so we do not expect candidates to come to us with knowledge of the above mentioned topics, but we do expect a strong interest in ontologies or algorithms and a desire to create good software. The Robinson lab is very collaborative, taking a team science approach to many projects and participating in efforts such as the [Global Alliance for Genomics and Health (GA4GH)](https://www.ga4gh.org/). We expect the candidate to participate in collaborations, which means concretely participating in virtual conferences and co-developing code using GitHub. Other duties include contributing to publications, submission of grants and funding applications, potential commercialization opportunities, as well as partnerships and collaborations with the community-wide efforts aimed at promoting open, integrated, and rigorous practices in research and scholarly communication. 

The Robinson lab develops software mainly in Java and Python but works with a range of other technologies such as SQL, JavaScript, C++, Rust, and R.

### Training

Lab members receive substantial training in software engineering and algorhthmics. In general, this takes place in the context 
of ad hoc meetings. The lab also has offered software practica (this year we will be offering a practicum in introduction to Rust programming). In addition, we have weekly group meetings. Generally, one-on-one meetings with Peter or mini-hackathons with small groups are held according to project needs (for instance, weekly). 
